packages/main/test/specs/TimePicker.spec.js

```diff
@@ -1,6 +1,7 @@
 import { assert } from "chai";
 
 describe("TimePicker general interaction", () => {
+
 	it("input receives value in format pattern depending on the set language", async () => {
 		await browser.url(`test/pages/TimePicker.html?sap-ui-language=bg`);
```

Comment:
We will probably refactor the exiting spec files to minimize the loading of the test page, that's why I am commenting the usage of `await browser.url` only in the newly added specs.

Reply:
Done.